### PR TITLE
Add core weekly 5 of 2020

### PR DESCRIPTION
**Pull request by matks**

### CODE REVIEW

File: news/_posts/core-weekly/2020-02-03-coreweekly-week-05-2020.md [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+
+This edition of the Core Weekly report highlights changes in PrestaShop's core codebase from Monday 27th of January to Sunday 2nd of February 2020.
+
+![Core Weekly banner](/assets/images/2018/12/banner-core-weekly.jpg)
+
+## General messages
+
+Dear developers,
+
+This week we [introduced a new tool](https://github.com/PrestaShop/PrestaShop/pull/16704) in our CI pipeline that we believe will make PrestaShop Javascript a lot easier to read and maintain: we now use [eslinter](https://eslint.org/) on the modern BO theme files. This popular tool will help us to ensure the monitored code is written in a consistent way, following popular and standard javascript conventions. It also looks for regular bugs and issues in the code and reports them at build time.
```
Comment (matks) [inline news/_posts/core-weekly/2020-02-03-coreweekly-week-05-2020.md]: @eternoendless @NeOMakinG @PierreRambaud please confirm this statement is accurate (I'm not a JS expert)

Comment (ttoine) [reply news/_posts/core-weekly/2020-02-03-coreweekly-week-05-2020.md]: "usual" instead of "regular"?

Comment (NeOMakinG) [reply news/_posts/core-weekly/2020-02-03-coreweekly-week-05-2020.md]:
```suggestion
This week we [introduced a new tool](https://github.com/PrestaShop/PrestaShop/pull/16704) in our CI pipeline that we believe will make PrestaShop Javascript a lot easier to read and maintain: we now use [ESLint](https://eslint.org/) on the modern BO theme files. This popular tool will help us to ensure the monitored code is written in a consistent way, following the airbnb's configuration which follow popular and standard javascript conventions. It also looks for possible bugs and issues in the code and reports them at build time.
```

Not sure about the syntax but it looks better, @PierreRambaud what do you think about this ?

Comment (eternoendless) [reply news/_posts/core-weekly/2020-02-03-coreweekly-week-05-2020.md]:
```suggestion
This week we [introduced a new tool](https://github.com/PrestaShop/PrestaShop/pull/16704) in our CI pipeline that we believe will make PrestaShop Javascript a lot easier to read and maintain: we now use [ESLint](https://eslint.org/) on the modern BO theme files. This popular tool will help us ensure the monitored code is written in a consistent way, following our [code conventions](https://devdocs.prestashop.com/1.7/development/coding-standards/#javascript-code-conventions). It also looks for possible bugs and issues in the code and reports them at build time.
```

**Comment by matks** [comment]
Thank you all for review

**Review by eternoendless — APPROVED** [review]

**Comment by eternoendless** [comment]
Thank you @matks

**Pull request merged**